src/steps/change_directory_step.go

```diff
@@ -27,7 +28,9 @@ func (step *ChangeDirectoryStep) Run() error {
 	_, err := os.Stat(step.Directory)
 	if err == nil {
 		script.PrintCommand("cd", step.Directory)
-		return os.Chdir(step.Directory)
+		if !dryrun.IsActive() {
```

Comment:
What exactly are `clean guard clauses`? Thinking on this more this one isn't needed?